debug_toolbar/decorators.py

```diff
@@ -11,7 +11,7 @@ def require_show_toolbar(view):
     def inner(request, *args, **kwargs):
         from debug_toolbar.middleware import get_show_toolbar
 
-        show_toolbar = get_show_toolbar()
```

Comment:
Shouldn't we pass `async_mode` based on the type of request `WSGIRequest` or `ASGIRequest` here ? I mean I am not sure whether `require_show_toolbar` would ever be exposed to a custom `async view` in async context or vice versa but this would save the conversion as it slows down  the overall process. thoughts?